crypto/dsa/dsa_local.h

```diff
@@ -14,7 +14,7 @@
 struct dsa_st {
     /*
      * This first variable is used to pick up errors where a DSA is passed
-     * instead of of a EVP_PKEY
+     * instead of a EVP_PKEY
```

Comment:
_an_ instead of _a_ here?

Reply:
of course. there are several more of these that i'll address in the update